include/boost/random/detail/vector_io.hpp

```diff
@@ -29,6 +29,7 @@ void print_vector(std::basic_ostream<CharT, Traits>& os,
         iter = vec.begin(),
         end =  vec.end();
     os << os.widen('[');
+    os << os.widen(' ');
```

Comment:
>Is it absolutely necessary to change the format? Have you at least 
checked that the original format is still readable by the new code.

Reverted, and fixed upstream in multiprecision.

John.